apache/superset
# feat(ssh_tunnel): SQLAlchemy Form UI

### SUMMARY
Add our a new `SSHTunnelForm` component that will be used to set tunnels for our Databases, right now only works when using SQLAlchemy URIs.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### TESTING INSTRUCTIONS
1. Make sure the SSH_TUNNELING feature flag is enabled
2. open the create database modal
3. click on SQL Alchemy link so the SQLA form is displayed
4. Enable the new SSH Tunnel switch
5. Make use of the form so you can connect a DB with a SSH Tunnel attached to it

### ADDITIONAL INFORMATION
